src/levelset/levelSetKernel.cpp

```diff
@@ -171,6 +171,79 @@ bool LevelSetKernel::intersectCellPlane( long id, const std::array<double,3> &ro
     return CGElem::intersectPlaneBox( root, normal, minPoint, maxPoint, dim, tolerance);
 }
 
+/*!
+ * Given a cell considered as a box, and one of its interfaces, considered as one fo the six box
```

Comment:
From the documentation it's not clear what this function does. How can you consider a tetrahedron a box?

Reply:
This function is removed.